draft-ietf-httpbis-digest-headers.md

```diff
@@ -252,23 +252,22 @@ The example below shows the  `sha-256` digest-algorithm which uses base64 encodi
 
 # The Digest Header Field {#digest-header}
 
-The Digest header field provides a digest of the representation data.
+The Digest header field contains a list of one or more representation digest values
+as defined in {{representation-digest}}.
```

Comment:
Moves the fragment sentence from below into this paragrah
```suggestion
as defined in {{representation-digest}}. It can be used in both request and response.
```